server/db/migrations/19_add_indexes_to_container_log.rb

```diff
@@ -1,5 +1,5 @@
 class AddIndexesToContainerLog < Mongodb::Migration
   def self.up
-    ContainerLog.create_indexes
+    Thread.new { ContainerLog.create_indexes } # might take a long time
```

Comment:
The `Thread.new { ... }` silently hides any exceptions... I'd have suggested a `Celluloid::Future`, but it also does that :/